src/util/fetchUtils.js

```diff
@@ -94,8 +98,17 @@ export const mergeCookies = (rawCookieHeader, newCookies) => {
 		...oldCookies,
 		...newCookies,
 	};
-	return Object.keys(mergedCookies)
-		.map(name => `${name}=${mergedCookies[name]}`)
-		.join('; ');
+	return stringifyCookies(mergedCookies);
+};
+
+export const BAD_COOKIES = [
```

Comment:
I agree. In this case, we need to pass along most of the cookies provided by Meetup Classic, but not this one. Without refactoring the clickTracking setup in Meetup Classic to ensure that it's sending 'officially valid' cookies, however, I think a blacklist is the best compromise solution.

If Meetup Classic adds more cookies that aren't valid in the future, then they will throw errors, although we now have better error logging to help us diagnose and fix the problem.

A whitelist might also work short-term, but it's hard to know when Meetup Classic might start requiring different cookies than what it currently sends.